exercises/05-reselect/src/modules/root/components/root.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import { Header } from './header'
+import { UserList } from 'modules/users/components/user-list'
+
+export const Root: React.FC = () => (
+  <div>
```

Comment:
If there's no styling reason. You can use `<></>` instead to not pollute DOM more than needed.

Reply:
Replaced here and in user-list component. In this and all previous exercises. See 3cd4424